pkg/engine/lifecycletest/test_plan.go

```diff
@@ -105,6 +106,13 @@ func (op TestOp) runWithContext(
 		BackendClient:   backendClient,
 	}
 
+	updateOpts := opts.Options()
+	defer func() {
+		if updateOpts.Host != nil {
+			contract.IgnoreClose(updateOpts.Host)
```

Comment:
Does this need to close? Doesn't `op` close this anyway, thus why we're doing this change.

Reply:
Yes, it is merely a defensive measure because here is the final opportunity to close the host before leaking it.